.travis.yml

```diff
@@ -0,0 +1,30 @@
+# This is the Travis-CI file for the pipe2d package.
+#
+# For a ".travis.yml" file for components of the PFS 2D pipeline
+# (as opposed to this top-level package which contains the tests),
+# see the ".travis.yml.TEMPLATE" file.
+
+language: python
+sudo: false
+python:
+  - 2.7
+#  - 3.5
+os:
+  - linux
+#  - osx
+matrix:
+  fast_finish: true
+notifications:
+  email: false
+
+install: true
+script: ./bin/pfs_travis.sh
+
+cache:
```

Comment:
Updated following your latest commit — I see the `find` has gone, but the “bleeding edge casher” doesn't seem to be documented at https://docs.travis-ci.com/user/caching, so I think maybe a little more explanation might still be helpful.